navigation/create_control/config/create_2_control.yaml

```diff
@@ -0,0 +1,43 @@
+# http://wiki.ros.org/diff_drive_controller for more parameters.
+
+mobile_base_controller:
+  type        : "diff_drive_controller/DiffDriveController"
+  left_wheel  : 'wheel_left_joint'
+  right_wheel : 'wheel_right_joint'
+  publish_rate: 50.0               # default: 50
+  pose_covariance_diagonal : [0.001, 0.001, 1000000.0, 1000000.0, 1000000.0, 1000.0]
+  twist_covariance_diagonal: [0.001, 0.001, 1000000.0, 1000000.0, 1000000.0, 1000.0]
+
+  # Wheel separation and diameter. These are both optional.
+  # diff_drive_controller will attempt to read either one or both from the
+  # URDF if not specified as a parameter
+  wheel_separation : 1.0
+  wheel_radius : 0.3
+
+  # Wheel separation and radius multipliers
+  wheel_separation_multiplier: 1.0 # default: 1.0
+  wheel_radius_multiplier    : 1.0 # default: 1.0
```

Comment:
Could you give me a pointer where to find it?

Reply:
You don't need to implement it in case you have access to a real robot, but in [this issue](https://github.com/RoboticaUtnFrba/create_autonomy/issues/57) you can find some material. And particularly for UMBmark, see [this](http://thetechnicgear.com/2014/06/howto-calibrate-differential-drive-robot/).

Reply:
Can you make a comment about what I told you here?